vortexor/src/main.rs

```diff
@@ -67,14 +100,51 @@ pub fn main() {
     )
     .unwrap();
 
-    // The _non_vote_receiver will forward the verified transactions to its configured validator
-    let (non_vote_sender, _non_vote_receiver) = banking_tracer.create_channel_non_vote();
+    let config = SocketConfig::default().reuseport(false);
+
+    let sender_socket =
+        bind_in_range_with_config(bind_address, dynamic_port_range, config).unwrap();
+
+    // The non_vote_receiver will forward the verified transactions to its configured validator
+    let (non_vote_sender, non_vote_receiver) = banking_tracer.create_channel_non_vote();
 
+    let destinations = values_t!(matches, "destination", String)
+        .unwrap_or_default()
+        .into_iter()
+        .map(|destination| {
+            solana_net_utils::parse_host_port(&destination).unwrap_or_else(|e| {
+                panic!("Failed to parse destination address: {e}");
```

Comment:
I think clapv4 can do much of this parsing and validation automatically.

Reply:
@alexpyattaev Can we defer clap upgrade to another PR as the core change of this PR is not about clap. I need to control the size of one PR.